<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>36</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Трекер события</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Описание события</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Следить</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>16</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>До наступления события осталось: ХХ дней</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
